public/index.hbs

```diff
@@ -29,7 +28,6 @@ with=""
 <div class="p-card u-no-padding">
     <div class="p-card__inner">
         <div class="p-media-object--large u-no-margin--bottom">
-            <img src="/resources/vrc-get.svg" class="p-media-object__image" alt="">
```

Comment:
I was mean left square blank for vrc-get and keep ALCOM logo but current design looks better! Thanks!